apps/outlook-rooms-addin/src/app/rooms/features-filter.service.ts

```diff
@@ -33,43 +39,55 @@ export class FeaturesFilterService {
         { name: 'Jamboard', id: 'Jamboard', value: false },
         { name: 'Projector', id: 'Projector', value: false },
         { name: 'Views', id: 'Views', value: false },
-    ]);
+    ];
 
     constructor(private _state: EventFormService) {
-        this.features$ = this.Features;
-        this._getSelectedFeatures();
+        this._features.next(this.room_features);
     }
 
-    private _getSelectedFeatures() {
-        this.selected_features$ = this.features$.pipe(
+    async getSelectedFeatures() {
+        this.selected_features$ = await this.features$.pipe(
```

Comment:
You should be converting this to a `Promise` if you want to use `await` with it. Observables have a `toPromise` method. Though you will to need to use `take(1)` operator as well, to convert open `Observable` to a `Promise` that resolves.